src/controllers/authController.js

```diff
@@ -0,0 +1,32 @@
+import passport from 'passport';
+import JWT from 'jsonwebtoken';
+import { config } from 'dotenv';
+import passportConfIgnored from '../config/passport';
```

Comment:
This package wasn't used

Reply:
It isn't a package. It's a configuration, and weirdly enough, passport won't work without it being imported this way. No other reference to it, just import it into the scope here.